### PR TITLE
Upgrade alfajores to op

### PR DESCRIPTION
**Pull request by aaronmgdr**
Alfajores will be upgraded from an L1 to an OP L2 at end of the month

This PR adds the relevant contract addresses to the chain definition 

(pending review by team)

---

## PR-Codex overview
This PR updates the Celo Alfajores chain configuration to include new contract addresses.

### Detailed summary
- Updated Celo Alfajores chain with new contract addresses
- Added contracts: portal, disputeGameFactory, l2OutputOracle, l1StandardBridge, l2StandardBridge

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/celoAlfajores.ts [hunk 1]
```diff
@@ -21,12 +23,43 @@ export const celoAlfajores = /*#__PURE__*/ defineChain({
       url: 'https://explorer.celo.org/alfajores',
       apiUrl: 'https://explorer.celo.org/api',
     },
+    celoscan: {
+      name: 'CeloScan',
+      url: 'https://alfajores.celoscan.io/',
+      apiUrl: 'https://api-alfajores.celoscan.io/api',
+    },
```
Comment (tmm) [inline src/chains/definitions/celoAlfajores.ts:30]: `blockExplorers` isn't meant to be exhaustive and we only support one per chain so can either make this the default or remove it.

Comment (aaronmgdr) [reply src/chains/definitions/celoAlfajores.ts:30]: cool

Comment (aaronmgdr) [reply src/chains/definitions/celoAlfajores.ts:30]: the type definition allows for other arbitrary keys, is the idea that users can add their own in in the app they are building?

File: src/celo/chainConfig.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
+import { contracts } from '../op-stack/contracts.js'
```
Comment (aaronmgdr) [inline src/celo/chainConfig.ts:1]: for easier review this is 

```typescript 

const contracts = {
  gasPriceOracle: { address: '0x420000000000000000000000000000000000000F' },
  l1Block: { address: '0x4200000000000000000000000000000000000015' },
  l2CrossDomainMessenger: {
    address: '0x4200000000000000000000000000000000000007',
  },
  l2Erc721Bridge: { address: '0x4200000000000000000000000000000000000014' },
  l2StandardBridge: { address: '0x4200000000000000000000000000000000000010' },
  l2ToL1MessagePasser: {
    address: '0x4200000000000000000000000000000000000016',
  },
} 
```

File: src/chains/definitions/celoAlfajores.ts (hunk 1 above)
Comment (tmm) [reply src/chains/definitions/celoAlfajores.ts]: exactly!

**Comment by aaronmgdr** [comment]
@jcortejoso please review addresses

**Comment by aaronmgdr** [comment]
@tmm I based the contracts on what base / op include in their config. there isn't any reason to have additional contract addresses defined right? 

slightly related but would it be correct to add additional celo core contracts to this config?

**Comment by tmm** [comment]
> there isn't any reason to have additional contract addresses defined right?

Correct, unless they are used internally in Viem. (Can search `chain.contracts` to see if you missed any.)

> slightly related but would it be correct to add additional celo core contracts to this config?

You are welcome to add additional ones that you know will be used downstream, but similar to block explorers/RPC urls, we try to keep things lean when we can!

**Pull request closed without merging**